client/marklogicClient.ts

```diff
@@ -407,3 +408,20 @@ export function newMarklogicManageClient(dbClientContext: ClientContext, manageP
     manageClientParams.modulesDb = null;
     return new ClientContext(manageClientParams);
 }
+
+export function newClientParamsFromWorkspaceConfigurationAndOverrides(cfg: WorkspaceConfiguration, overrides: object = {}): MlClientParameters {
```

Comment:
I think `newClientParams` is fine here, until there's another instance of the function that would require a name that distinguishes it from that other function. But we can cross that bridge later.